src/components/Calculator.js

```diff
@@ -1,13 +1,14 @@
+/* eslint-disable import/extensions */
 import React, { useState } from 'react';
-import '../calculator.css';
-import calculate from '../logic/calculate';
+import calculate from '../Logic/calculate';
```

Comment:
- kindly, your directory name is missing so it will not run on most environments :+1:  you have to be sure also its case also it will be `../logic/calculate` :+1:

Reply:
please could you clarify this change. because I have changed the file name everywhere and the project is displaying on all machines therefore, I NEED CLARITY ON THIS.

Reply:
@Moise-code , your Directory written in begin with small letters `logic` but you are accessing with upper case `Logic`

Reply:
could you check for me, I have changed it.

Reply:
Thank you so much I appreciate it.

Reply:
though the status on dashboard it is still marking required changes @binhussen